src/main.ts

```diff
@@ -23,65 +22,42 @@ export async function run(): Promise<void> {
       urlReplacement,
       githubComment,
       githubToken,
-      runInAsyncMode,
-      testGroupId
+      testSuiteId,
+      runInAsyncMode
     } = parseInput();
 
-    const httpClient = new HttpClient('stably-runner-action', [
-      new BearerCredentialHandler(apiKey)
-    ]);
-    const respPromise = fetchSSE({
-      httpClient,
-      url: 'https://app.stably.ai/api/runner/run',
-      payload: {
-        testGroupId,
-        ...(urlReplacement ? { domainOverrides: [urlReplacement] } : {})
-      }
-    });
```

Comment:
I see you refactored the HTTP call out into it's own file, I don't think we should do that.

  1. The API call is only used one place (YAGNI)
  2. This file, even with the HTTP call is less than 100 lines
  3. Putting it another file (`api.ts`) forces you to re-define types and to have an extra if clause. You're adding about 30-40 lines of extra code that's a pain to maintain

Reply:
Discussed during our call.